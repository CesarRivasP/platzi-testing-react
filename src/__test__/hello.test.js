
import React from 'react';
import Hello from '../components/hello';
import Enzyme, { mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });


describe('AÃ±adiendo pruebas a React', () => {
  test('Comprobamos el texto que recibe la prop', () => {
    const data = {
      title: 'Hello Jest'
    };

    const wrapper = mount(
      <Hello title={data.title} />
    );

    const htmlTag = wrapper.find('h1');

    expect(htmlTag.text()).toBe('Hello Jest');
  });

  test('Creamos un snapshot en React', () => {
    const data = { 
      title: 'Hello Jest'
    };

    const wrapper = shallow(
      <Hello title={data.title} />
    );

    expect(wrapper).toMatchSnapshot();
  });
});