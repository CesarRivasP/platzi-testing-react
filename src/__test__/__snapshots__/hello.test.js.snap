// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AÃ±adiendo pruebas a React Creamos un snapshot en React 1`] = `ShallowWrapper {}`;
